ui/app/styles/components/linked-block.scss

```diff
@@ -1,13 +1,11 @@
 .linked-block {
   cursor: pointer;
+  transition: box-shadow $speed;
```

Comment:
probably want to pair this with a `will-change: box-shadow;`

Reply:
I always forget about that. Done!